backend/src/main/scala/cromwell/backend/standard/callcaching/StandardFileHashingActor.scala

```diff
@@ -63,12 +65,10 @@ abstract class StandardFileHashingActor(standardParams: StandardFileHashingActor
   def fileHashingReceive: Receive = {
     // Hash Request
     case fileRequest: SingleFileHashRequest =>
-      val replyTo = sender()
-
       customHashStrategy(fileRequest) match {
         case Some(Success(result)) => context.parent ! FileHashResponse(HashResult(fileRequest.hashKey, HashValue(result)))
         case Some(Failure(failure)) => context.parent ! HashingFailedMessage(fileRequest.file.value, failure)
-        case None => asyncHashing(fileRequest, replyTo)
+        case None => asyncHashing(fileRequest, context.parent)
```

Comment:
Conversation with @Horneth says that this actor should be consistent, and always talk to the `context.parent`, not sometimes the `sender()`.